iaso/api/storage.py

```diff
@@ -193,27 +246,37 @@ def list(self, request):
         page_offset = request.GET.get("page", 1)
         order = request.GET.get("order", "updated_at,id").split(",")
 
+        csv_format = request.GET.get("csv", None)
+        xlsx_format = request.GET.get("xlsx", None)
+        file_export = False
+        if csv_format is not None or xlsx_format is not None:
+            file_export = True
+            file_format_export = FileFormatEnum.CSV if csv_format is not None else FileFormatEnum.XLSX
+
         queryset = self.get_queryset()
         queryset = queryset.order_by(*order)
         serializer = StorageSerializer
 
-        if limit_str is not None:
-            limit = int(limit_str)
-            page_offset = int(page_offset)
-            paginator = Paginator(queryset, limit)
-            res = {"count": paginator.count}
-            if page_offset > paginator.num_pages:
-                page_offset = paginator.num_pages
-            page = paginator.page(page_offset)
-            res["results"] = serializer(page.object_list, many=True).data
-            res["has_next"] = page.has_next()
-            res["has_previous"] = page.has_previous()
-            res["page"] = page_offset
-            res["pages"] = paginator.num_pages
-            res["limit"] = limit
-            return Response(res)
-        else:
-            return Response(StorageSerializer(queryset, many=True).data)
+        if not file_export:
```

Comment:
I don't like : 

```python
if not XXXX:
   A
else:
   B

I prefer
if XXXX:
   B
else:
   A
```

What do you think ?

Reply:
Thanks, I don't have a strong opinion for the matter, and in that case used the construct so the more common case comes first. Anyway, I changed it according to your suggestion.

Reply:
It's really a matter of taste for this one I guess